packages/p2p/src/components/my-ads/my-ads-row-renderer.jsx

```diff
@@ -33,13 +34,15 @@ const MyAdsRowRenderer = observer(({ row: advert, setAdvert }) => {
     const [is_popover_actions_visible, setIsPopoverActionsVisible] = React.useState(false);
 
     const amount_dealt = amount - remaining_amount;
+    const enable_action_point = rate_type === 'fixed' && floating_rate_store.change_ad_alert;
```

Comment:
Let's create constants out of fixed and float.

Reply:
Ok. Will make this change